.github/workflows/e2e-test-docker-image-build.yml

```diff
@@ -10,14 +16,25 @@ permissions:
 jobs:
   build-and-push-image:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        canary: [ { repo: 'java-eks', dir: '/terraform/java/eks' },
+                  { repo: 'java-ec2-default', dir: '/terraform/java/ec2/default' },
+                  { repo: 'java-ec2-asg', dir: '/terraform/java/ec2/asg' },
+                  { repo: 'java-k8s', dir: '/terraform/java/k8s' },
+                  { repo: 'java-metric-limiter', dir: '/terraform/java/eks' },
+                  { repo: 'python-eks', dir: '/terraform/python/eks' },
+                  { repo: 'python-ec2-default', dir: '/terraform/python/ec2/default' },
+                  { repo: 'python-ec2-asg', dir: '/terraform/python/ec2/asg' },
+                  { repo: 'python-k8s', dir: '/terraform/python/k8s' } ]
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
 
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
-          role-to-assume: ${{ secrets.E2E_SECRET_TEST_ROLE_ARN }}
+          role-to-assume: ${{ secrets.E2E_IAD_TEST_ACCOUNT_ARN }}
```

Comment:
The value changed from the pipeline account to the IAD test account, right?

Reply:
Yes